app/code/core/Mage/Adminhtml/Block/Review/Grid.php

```diff
@@ -192,7 +192,20 @@ protected function _prepareColumns()
                 'sortable'  => false
             ]);
 
-        $this->addRssList('rss/catalog/review', Mage::helper('catalog')->__('Pending Reviews RSS'));
+        if (Mage::helper('catalog')->isModuleEnabled('Mage_Rss')) {
+            if ($this->getColumn('visible_in')) {
+                $filter = $this->getParam($this->getVarNameFilter());
+                if (!empty($filter)) {
+                    $filter = Mage::helper('adminhtml')->prepareFilterString($filter);
+                    if (!empty($filter['visible_in']))
+                        $storeId = $filter['visible_in'];
+                }
+            }
+            if (empty($storeId))
```

Comment:
Reviewed a few commits ... can you please avoid `empty()` when not required and use more strict checks?

```
                $filter = $this->getParam($this->getVarNameFilter());
                if ($filter !== '') {
                    $filter = Mage::helper('adminhtml')->prepareFilterString($filter);
                    if (isset($filter['visible_in']))
                        $storeId = $filter['visible_in'];
                }
            }
            if ($storeId === null) # storeId should be defined above
```

Not tested.